db/structure.sql

```diff
@@ -792,7 +792,7 @@ CREATE TABLE public.donations (
     card_id integer,
     designation text,
     offsite boolean,
-    anonymous boolean,
+    anonymous boolean DEFAULT false NOT NULL,
```

Comment:
I think you might have an old migration still in place! This shouldn't be `NOT NULL.` To cleanup, clear your db,  and update from a recent database.

`bin/rake db:drop db:create`
`script/restore_from_heroku.sh` or `script/pg_restore_local_from_production.sh`